src/pages/home/report/ReportActionsView.js

```diff
@@ -118,36 +115,14 @@ function ReportActionsView(props) {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
-    useEffect(() => {
-        const prevNetwork = prevNetworkRef.current;
-        // When returning from offline to online state we want to trigger a request to OpenReport which
-        // will fetch the reportActions data and mark the report as read. If the report is not fully visible
-        // then we call ReconnectToReport which only loads the reportActions data without marking the report as read.
-        const wasNetworkChangeDetected = lodashGet(prevNetwork, 'isOffline') && !lodashGet(props.network, 'isOffline');
-        if (wasNetworkChangeDetected) {
-            if (isReportFullyVisible) {
-                openReportIfNecessary();
-            } else {
-                Report.reconnect(reportID);
-            }
-        }
-        // update ref with current network state
-        prevNetworkRef.current = props.network;
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [props.network, props.report, isReportFullyVisible]);
 
     useEffect(() => {
         const wasLoginChangedDetected = prevAuthTokenType === 'anonymousAccount' && !props.session.authTokenType;
         if (wasLoginChangedDetected && didUserLogInDuringSession() && isUserCreatedPolicyRoom(props.report)) {
-            if (isReportFullyVisible) {
-                openReportIfNecessary();
-            } else {
-                Report.reconnect(reportID);
-            }
+            openReportIfNecessary();
```

Comment:
This is the only case we still need to fetch report with OpenReport because this is not related to loosing/regaining connection, so `ReconnectApp` is not called.

In the past, we were call `Report.reconnect(reportID);` instead of `openReportIfNecessary();` if the App is in the background/not focused, but I don't think it is important to have this consideration here because this case only happens when we are login in from an anonymous account, and we can't log in with the app in the background.